src/cpl/lilac/lnd_comp_esmf.F90

```diff
@@ -626,106 +625,106 @@ subroutine lnd_run(gcomp, import_state, export_state, clock, rc)
 
-       !--------------------------------
-       ! Determine doalb based on nextsw_cday sent from atm model
-       !--------------------------------
+    if (masterproc) then
+       write(iulog,*) '~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~'
+       write(iulog,F02) 'nextsw_cday is :  ', nextsw_cday
+       write(iulog,F02) 'calday is      :  ', calday
+       write(iulog,F02) 'eccen is       :  ', eccen
+       write(iulog,F02) 'mvelpp is      :  ', mvelpp
+       write(iulog,F02) 'lambm0 is      :  ', lambm0
+       write(iulog,F02) 'obliqr is      :  ', obliqr
+       write(iulog,F02) 'declin is      :  ', declin
+       write(iulog,F02) 'declinp1 is    :  ', declinp1
+       write(iulog,*  ) '~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~'
+    end if
 
-       if (nstep == 0) then
-          doalb = .false.
-          nextsw_cday = caldayp1
-       else if (nstep == 1) then
-          !doalb = (abs(nextsw_cday- caldayp1) < 1.e-10_r8)
-          doalb = .false.
-       else
-          doalb = (nextsw_cday >= -0.5_r8)
-       end if
+    !--------------------------------
+    ! Determine doalb based on nextsw_cday sent from atm model
+    !--------------------------------
 
-       if (masterproc) then
-          write(iulog,*) '------------  LILAC  ----------------'
-          write(iulog,*) 'nstep       : ', nstep
-          write(iulog,*) 'calday      : ', calday
-          write(iulog,*) 'caldayp1    : ', caldayp1
-          write(iulog,*) 'nextsw_cday : ', nextsw_cday
-          write(iulog,*) 'doalb       : ', doalb
-          write(iulog,*) '-------------------------------------'
-       end if
+    if (nstep == 1) then
+       !doalb = (abs(nextsw_cday- caldayp1) < 1.e-10_r8)
+       doalb = .false.
+    else
+       doalb = (nextsw_cday >= -0.5_r8)
+    end if
+
+    if (masterproc) then
+       write(iulog,*) '------------  LILAC  ----------------'
+       write(iulog,*) 'nstep       : ', nstep
+       write(iulog,*) 'calday      : ', calday
+       write(iulog,*) 'caldayp1    : ', caldayp1
+       write(iulog,*) 'nextsw_cday : ', nextsw_cday
+       write(iulog,*) 'doalb       : ', doalb
+       write(iulog,*) '-------------------------------------'
+    end if
 
-       call update_rad_dtime(doalb)
+    call update_rad_dtime(doalb)
 
+<<<<<<< HEAD
        !--------------------------------
        ! Determine if time to stop
        !--------------------------------
 
        call ESMF_ClockGetAlarm(clock, alarmname='lilac_stop_alarm', alarm=alarm, rc=rc)
+=======
+    !--------------------------------
+    ! Determine if time to write restart
+    !--------------------------------
+
+    call ESMF_ClockGetAlarm(clock, alarmname='lilac_restart_alarm', alarm=alarm, rc=rc)
+    if (ChkErr(rc,__LINE__,u_FILE_u)) return
+
+    if (ESMF_AlarmIsRinging(alarm, rc=rc)) then
        if (ChkErr(rc,__LINE__,u_FILE_u)) return
+       rstwr = .true.
+       call ESMF_AlarmRingerOff( alarm, rc=rc )
+>>>>>>> ctsm5.3.019
```

Comment:
Please resolve this conflict with the 019 version.

Reply:
@ekluzek suggests reordering the alarms after picking the 019 code, to completely resolve these two conflicts.

Reply:
The purpose of the #2912 was to reverse the order that the alarms were set in.

So change it to the ctsm5.3.019 way as @slevis-lmwg points out and then reverse the order of how the alarms are set, which is all that #2912 was doing.